docs/source/quickstart.mdx

```diff
@@ -75,7 +68,7 @@ Static quantization relies on feeding batches of data through the model to estim
 >>> calibration_dataset = quantizer.get_calibration_dataset(
 ...     "glue",
 ...     dataset_config_name="sst2",
-...     preprocess_function=partial(preprocess_fn, tokenizer=quantizer.preprocessor),
+...     preprocess_function=partial(preprocess_fn, tokenizer=tokenizer),
```

Comment:
I think we're still missing the `preprocess_fn`. Should we add the following above?

```
# Define the processing function to apply to each example after loading the dataset
def preprocess_fn(ex, tokenizer):
    return tokenizer(ex["sentence"])
```

Reply:
You're right, it was missing in the documentation quickstart, I just added it !

Reply:
Great!